src/main/resources/schema/provider.definition.schema.v1.json

```diff
@@ -304,7 +304,7 @@
         "taggable": {
             "$comment": "A boolean flag indicating whether this resource type supports updatable tagging.",
             "type": "boolean",
-            "default": true
+            "default": "Default to true if there's an UpdateHandler, and false if no UpdateHandler is provided."
```

Comment:
maybe just remove `default` since I think it's for literal values, but this would be a good `description` to add
https://json-schema.org/understanding-json-schema/reference/generic.html

Reply:
Sure nice catch